.github/workflows/check_format.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Check include guards
         if: always()
         run: |
-          find include test -type f '(' -name "*.h" -o -name "*.h.in" ')'             \
+          find . test -type f '(' -name "*.h" -o -name "*.h.in" ')'             \
```

Comment:
minor: can you fix the alignment of the line continuation backslash?

Reply:
Yep, good catch: https://github.com/eth-cscs/DLA-Future/pull/1117/commits/cc12ac586e0a6eb6855ad27b5ee870a4d3c72387.